src/OpenTelemetry/README.md

```diff
@@ -308,6 +305,21 @@ Core, Console.
 }
 ```
 
+Note: "Current directory" of your process is the working directory of your
+process. It might be different depending on how you run the program.
+
+For example, for [AspNetCore example project](../../examples/AspNetCore), if
```

Comment:
I'm not sure if this section is going to be helpful. Is there any feedback on this from the reader?

Reply:
We need an actual example for the common project types - Asp.Net, Asp.Net Core. There were several conversation around "where to place the file".

Reply:
As in the [description I removed](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1717/commits/f8546d3e4502ebccbefc1ead4c1e29e84fb88c8b), the current working directory might be different depending on how it is run (it would be project folder if compiling and running by project with `dotnet` command or Visual Studio, or executable folder if running the executable). And if running within an IIS server, it might be a different folder.

Which way of running shall we give the example? Or we can mention that they can figure out by calling [Directory.GetCurrentDirectory](https://docs.microsoft.com/dotnet/api/system.io.directory.getcurrentdirectory) method in their code?

Reply:
The suggestion of using Directory.GetCurrentDirectory is great.
I'd be happy to see specific examples for an Asp.Net Core , Asp.Net project as well, when they are run outside from Visual Studio F5 method, as thats the common production deployment.